index.html

```diff
@@ -1704,14 +1721,14 @@
         <h2>
           <dfn data-lt="complete(result)">complete()</dfn> method
         </h2>
-        <p>
+        <p class="note">
           The <a>complete()</a> method is called after the user has accepted
           the payment request and the <a>[[\acceptPromise]]</a> has been
           resolved. Calling the <a>complete()</a> method tells the <a>user
-          agent</a> that the user interaction is over (and SHOULD cause any
+          agent</a> that the user interaction is over (and can cause any
           remaining user interface to be closed).
         </p>
-        <div class="note">
+        <div class="note" title="Dealing with blocked browser UI">
```

Comment:
Agree. I've changed the second "may" to an RFC2119 "MAY" and un-noted these two paragraphs.